mesh/ncmesh.cpp

```diff
@@ -145,6 +145,7 @@ NCMesh::NCMesh(const Mesh *mesh)
       // if we have pyramids we will need tets after refinement
       if (geom == Geometry::PYRAMID)
       {
+         CheckSupportedGeom(Geometry::TETRAHEDRON);
```

Comment:
For an unknown variable, it makes sense to call `CheckSupportedGeom`, but it seems unnecessary to call it for a hard-coded value.